include/structs/dungeon_entity.h

```diff
@@ -112,57 +218,19 @@ typedef struct EntityInfo
     u8 fillA5[0xA8 - 0xA5];
     // Statuses are split into groups based on which ones can't overlap.
     // See status.h for which statuses are in each group.
-    /* 0xA8 */ u8 sleep;
-    /* 0xA9 */ u8 sleepTurns;
-    u8 fillAA[0xAC - 0xAA];
-    /* 0xAC */ u8 nonVolatileStatus;
-    /* 0xAD */ u8 nonVolatileStatusTurns;
-    /* 0xAE */ u8 nonVolatileStatusDamageCountdown;
-    u8 fillAF;
-    /* 0xB0 */ u8 immobilizeStatus;
-    u8 fillB1[0xB4 - 0xB1];
-    /* 0xB4 */ s32 unkB4;
-    /* 0xB8 */ u8 immobilizeStatusTurns;
-    /* 0xB9 */ u8 immobilizeStatusDamageCountdown;
-    u8 fillBA[0xBC - 0xBA];
-    /* 0xBC */ u8 volatileStatus;
-    /* 0xBD */ u8 volatileStatusTurns;
-    u8 fillBE[0xC0 - 0xBE];
-    /* 0xC0 */ u8 chargingStatus;
-    /* 0xC1 */ u8 chargingStatusTurns;
-    /* 0xC2 */ u8 chargingStatusMoveIndex; // The position of the move in the Pokémon's moveset that triggered the status.
-    u8 fillC3;
-    /* 0xC4 */ u8 protectionStatus;
-    /* 0xC5 */ u8 protectionStatusTurns;
-    u8 fillC6[0xC8 - 0xC6];
-    /* 0xC8 */ u8 waitingStatus;
-    /* 0xC9 */ bool8 enemyDecoy; // True if the Pokémon is a decoy and a wild Pokémon (i.e., not an allied Pokémon).
-    u8 fillCA;
-    /* 0xCB */ u8 waitingStatusTurns;
-    /* 0xCC */ u8 curseDamageCountdown;
-    u8 fillCD[0xD0 - 0xCD];
-    /* 0xD0 */ u8 linkedStatus;
-    u8 fillD1[0xD4 - 0xD1];
-    /* 0xD4 */ u32 unkD4;
-    /* 0xD8 */ u8 unkD8;
-    /* 0xD9 */ u8 linkedStatusTurns;
-    /* 0xDA */ u8 linkedStatusDamageCountdown;
-    u8 fillDB;
-    /* 0xDC */ u8 moveStatus;
-    /* 0xDD */ u8 moveStatusTurns;
-    u8 fillDE[0xE0 - 0xDE];
-    /* 0xE0 */ u8 itemStatus;
-    u8 fillE1[0xE4 - 0xE1];
-    /* 0xE4 */ u8 transformStatus;
-    /* 0xE5 */ u8 transformStatusTurns;
-    u8 fillE6[0xE8 - 0xE6];
-    /* 0xE8 */ u8 eyesightStatus;
-    /* 0xE9 */ u8 eyesightStatusTurns;
-    /* 0xEA */ u8 unkEA;
-    u8 fillEB;
-    /* 0xEC */ bool8 muzzled;
-    /* 0xED */ u8 muzzledTurns;
-    u8 fillEE[0xF0 - 0xEE];
+    /* 0xA8 */ Sleep sleep;
+    /* 0xAC */ NonVolatile nonVolatile;
+    /* 0xB0 */ Immobilize immobilize;
+    /* 0xBC */ Volatile Volatile;
```

Comment:
This var name should be lowercase to match the others.

Reply:
Can't name to lower case volatile because it's compiler keyword and will get compiler errors. I tried it once already.

Reply:
Oh right, perhaps `volatileStatus` or `volatileStruct` then?

Reply:
Named to `volatileStatus`